docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -199,6 +254,38 @@ def antarctica_schedule():
 
 Refer to the [Schedules documentation](/concepts/partitions-schedules-sensors/schedules#schedules-from-partitioned-assets-and-jobs) for more info about constructing both schedule types.
 
+## Partitioned Asset Job
+
+A partitioned job is a job that materializes a particular set of partitioned assets every time it runs.
```

Comment:
Should this sentence be `A partitioned asset job is a job...`?

This current iteration strikes me as odd because the previous section is called `Partitioned Jobs`. Based on this sentence I'm not sure what the difference is.

Reply:
great point. fixed